docs/source/seth_developers_guide/getting_started.rst

```diff
@@ -77,41 +73,38 @@ In order to interact with Seth, we need to create an external account on the
 network. Creating an account is equivalent to generating a new private key that
 Seth can understand. Seth accepts secp256k1 private keys in one of two formats:
 
-- A plaintext base58 encoded private key, such as those generated by
+- A plaintext hex encoded private key, such as those generated by
   ``sawtooth keygen``.
 - A PEM encoded private key with or without a password, such as those generated
   with the ``openssl`` command.
 
 The following shows you how to generate a PEM encoded key that has been
-encrypted with a password. If you would like to use a private key generated with
-the ``sawtooth keygen`` command, it must be copied from the ``validator``
-container where it was created.
+encrypted with a password.
 
-OpenSSL is already installed in the Docker container running the Seth-RPC
-server, so we can just connect to the container and run the command from there.
-Do the following to start up a shell in that container::
+OpenSSL is already installed in the ``seth-cli`` Docker container, so we can just
+connect to the container and run the command from there. Do the following to
+start up a shell in that container::
 
-    $ docker exec -it seth-rpc bash
+    $ docker exec -it seth-cli bash
 
 You should now be logged in as root in the container. From this shell, you can
 generate a new, password-encrypted key with the following::
 
-    # openssl ecparam -genkey -name secp256k1 | openssl ec -out alias.pem -aes128
+    # openssl ecparam -genkey -name secp256k1 | openssl ec -out key-file.pem -aes128
 
 You can use any encryption cipher by changing the final ``-aes128`` flag or omit
-the flag and to generate a key without it being encrypted. Note that the name of
-the key file must be the same as the alias the account is imported as.
+the flag and to generate a key without it being encrypted.
 
 Now we are ready to set up the account on the network. To do this, we need to
 use the ``seth`` command. From the prompt where you generated the key, run::
 
-    # seth account import alias.pem alias
+    # seth account import key-file.pem myalias
```

Comment:
No, they need to be the same, unless you've changed the way this works? so `{alias}.pem {alias}`